ui/apps/platform/src/Containers/VulnMgmt/List/Nodes/VulnMgmtListNodes.js

```diff
@@ -68,7 +68,7 @@ export function getNodeTableColumns(showVMUpdates) {
                 sortField: nodeSortFields.NODE,
             },
             {
-                Header: `CVEs`,
+                Header: `Node CVEs`,
```

Comment:
I think we need to feature flag the name, the entityType line, and the link building logic below ,for this.